libbeat/common/field.go

```diff
@@ -34,12 +34,15 @@ type Field struct {
 	DocValues      *bool       `config:"doc_values"`
 
 	// Kibana specific
-	Pattern      string `config:"pattern"`
 	Analyzed     *bool  `config:"analyzed"`
 	Count        int    `config:"count"`
 	Searchable   *bool  `config:"searchable"`
 	Aggregatable *bool  `config:"aggregatable"`
-	InputFormat  string `config:"input_format"`
+	Script       string `config:"script"`
+	Language     string `config:"lang"`
```

Comment:
I would suggest we only support painless script for now so we could hard code this.

Reply:
changed

Reply:
Do we still need this entry if it is always `painless`